scss/event.scss

```diff
@@ -1,47 +1,133 @@
 @import
-    'settings/colors',
-    'settings/variables',
-    'settings/mixins',
+'settings/colors',
+'settings/variables',
+'settings/mixins',
 
-    'modules/reset',
-    'modules/typography',
-    'modules/layout';
+'modules/reset',
+'modules/typography',
+'modules/layout';
 
 /**
  * Template specific classes
  */
 
 .event {
-    /**
-     * Event-headers
-     */
-    main > header {
-        background-position:center center;
-        background-size:cover;
+  /**
+   * Event-headers
+   */
+  main > header {
+    background-position:center center;
+    background-size:cover;
+  }
+
+  article {
+    margin:0;
+    padding:0;
+    box-shadow: none;
+    border: 0;
+    border-bottom: 2px solid $eventBorderColor;
+  }
+  .post-navigation {
+    @extend .sparks;
+    article a {
+      display: block;
     }
+  }
+
+}
+
+
+[class^="item-"] {
+  display:block;
+  position:relative;
+  margin:0;
+  padding: 2.4rem 1.48331rem;
+  border-radius: 0;
+
+  &,
+  &:hover,
+  &:active,
+  &:focus {
+    background: $eventBgColor;
+    text-decoration: none;
+    color: $textColor;
+  }
+
+  & + & {
+    border-top: 2px solid $eventBorderColor;
+  }
+
+  img {
+    display: block;
+    max-width: 100%;
+    height: auto;
+  }
+  h3 {
+    margin: 0 0 5px;
```

Comment:
Yes, we should probably invest the time to describe the basic principles in this repo. I'll see if I can find sometime today already.